app/forms/OverseasCompanyRegistrationForm.scala

```diff
@@ -22,19 +22,31 @@ import play.api.data.Forms._
 import play.api.i18n.Messages
 import play.api.i18n.Messages.Implicits._
 import play.api.Play.current
+import uk.gov.hmrc.play.mappers.StopOnFirstFail
+import uk.gov.hmrc.play.mappers.StopOnFirstFail.constraint
 
 object OverseasCompanyRegistrationForm {
 
   private val length60 = 60
   private val length40 = 40
+  private val idNumberRegex = "^[a-zA-Z0-9 '&\\-]{1,60}$"
+  private val issuingInstitutionRegex = "^[a-zA-Z0-9 '&\\-\\/]{1,40}$"
+  private val issuingCountryRegex = "(?!^GB$)^[A-Z]{2}$"
```

Comment:
Please follow best practices to name constants. Would it be worth moving these to constant file?